packages/nocodb/src/lib/elitesoftwareautomation/db/PSQLRecordOperationWatcher.ts

```diff
@@ -42,7 +42,7 @@ const regexForCharactersToReplace = /-/g;
 export class PSQLRecordOperationWatcher extends EventEmitter {
   public readonly recordOperationEventType = 'record-operation';
   private readonly rewatchErrorRetryDelayMillis = 3000;
-  private readonly throttleDelayMillis = 5000;
+  private readonly throttleDelayMillis = process.env.THROTTLE_DELAY_MILLIS || 5000;
```

Comment:
@horpeazy  env variables will be read as a string, you need to use parsInt or something similar here to convert it to a Number

Reply:
@GAMHnile I've corrected this